userguide/sharing.rst

```diff
@@ -1093,16 +1093,15 @@ settings:
   :menuselection:`Services --> SMB`.
 
 * When the :guilabel:`Browsable to Network Clients` box is checked (the 
-  default), the share will be visible in the server's browse list. If the share 
-  is is flagged as a :guilabel:`home share`, unchecking this box will hide the 
-  [homes] share so that only the dynamically generated home share with the
-  authenticated user's name will be visible (in contrast with [homes]
-  and [<username>] both being visible). 
-
-  The presence of a share in a user's browse list does not grant the user  
-  read or write permissions on the share. Users are still able to directly 
-  access and map shares that are not visible in the browse list by using 
-  the share's UNC path. Hence, this option is of limited security utility.
+  default), the share is visible when the user lists the available shares 
+  through Windows File Explorer or through the *net view* command. If the  
+  share is flagged as a :guilabel:`home share`, unchecking this box hides the 
```

Comment:
guilabel is for a GUI element like a button or list.  I'm not sure what "home share" represents here.  Well, a concept, but not a literal button.  I think.